src/reformat/rgb.rs

```diff
@@ -91,7 +91,7 @@ pub struct Image {
     pub format: Format,
     pub chroma_upsampling: ChromaUpsampling,
     pub chroma_downsampling: ChromaDownsampling,
-    pub alpha_premultiplied: bool,
+    pub premultiply_alpha: bool,
```

Comment:
Just so I understand, is this a command to premultiply the alpha or the state of the image?

Reply:
this is an input parameter that tells the conversion functions whether or not to premultiply the alpha.

once the conversion function is called, this also reflects the state of the image. ideally we should be splitting the parameters and the image state. it's on my list, will do it later.